test/recipes/80-test_ssl_old.t

```diff
@@ -811,20 +811,21 @@ sub testssl {
 	plan tests => 3;
 
       SKIP: {
-	  skip "Certificate Transparency is not supported by this OpenSSL build", 3
-	      if $no_ct;
-	  skip "TLSv1.0 is not supported by this OpenSSL build", 3
-	      if $no_tls1;
-
-    $ENV{CTLOG_FILE} = srctop_file("test", "ct", "log_list.conf");
-	  ok(run(test([@ssltest, "-bio_pair", "-tls1", "-noct"])));
-	  ok(run(test([@ssltest, "-bio_pair", "-tls1", "-requestct"])));
-	  # No SCTs provided, so this should fail.
-	  ok(run(test([@ssltest, "-bio_pair", "-tls1", "-requirect",
-	               "-should_negotiate", "fail-client"])));
-	}
+        skip "Certificate Transparency is not supported by this OpenSSL build", 3
```

Comment:
I'll look into it, but I'm pretty short on time so no promises :( The docs and API changes raised in this PR are my priority, once I find the time to return to OpenSSL work. I'm working on Chromium at the moment, which branches tomorrow, after which I'll switch back to this.